.gitignore

```diff
@@ -49,6 +49,7 @@ dump.rdb
 .DS_Store
 .ruby-gemset
 *.pdf
+!spec/fixtures/files/*.pdf
```

Comment:
Why are you adding this if you've added these files? Shouldn't they be tracked?

Reply:
line 51 tells git to ignore all .pdf files so line 52 specifies an exception (by prefixing an exclamation mark) to track the pdf files in spec/fixtures/files